x-pack/plugins/cases/public/common/mock/test_providers.tsx

```diff
@@ -46,6 +46,8 @@ export const useFormFieldMock = <T,>(options?: Partial<FieldHook<T>>): FieldHook
     type: 'type',
     value: ('mockedValue' as unknown) as T,
     isPristine: false,
+    isDirty: false,
```

Comment:
@elastic/kibana-security I would be interested to understand why you need this mock of the `FieldHook` to see if maybe there is another way to test your components. We haven't needed it in our tests so I am curious to know about your use case.

Reply:
I believe you wanted to ping @elastic/security-threat-hunting instead :slightly_smiling_face: